spanish-translations.md

```diff
@@ -0,0 +1,85 @@
+# Automatic Spanish Translations
+
+We're happy to announce that from now on we'll deliver Spanish translations to our new
+tutorials (scripts and subtitles). How did we make this possible? You might think that
+we hired a professional who takes care of translating the content. As great as human beings, 
+I'm proud to say that you would be wrong! As programmers, we love to automate processes, so, that's
+exactly what we did! We leverage the powerful translations service of *DeepL*, which is
+a translation service based on an AI algorithm. And... it's surprisingly good! But, 
+this time it wasn't as easy as making an API call, we had to heavily customize our 
+content so that some technical words or code blocks won't get translated and also keep
+the translation as accurate as possible. 
+
+## DeepL API
+
+DeepL allows you to wrap your content with XML tags to give it a special meaning, for example, 
```

Comment:
Maybe we can start this section with a short tutorial example - e.g.:

To understand the challenge, let's look at an example from our Doctrine tutorial:

```
For example, one command is called `doctrine:database:create`. Cool, let's try it: 

'''terminal
php bin/console doctrine:database:create
'''
```

Hmm. We want Deeply to translate the first two sentences, but *not* the technical word `doctrine:database:create`. We also want it to skip the `terminal` block entirely... and to just return it "verbatim" in the correct position. See, tricky!